package.json

```diff
@@ -8,7 +8,8 @@
     "build": "babel src -s -D -d dist --presets es2015,stage-0",
     "start": "node dist",
     "prestart": "npm run -s build",
-    "test": "eslint src"
+    "lint": "eslint src",
+    "precommit": "npm run lint & newman run https://www.getpostman.com/collections/cbc2e276cb8ce741a311"
```

Comment:
don't use this https://www.getpostman.com/collections/cbc2e276cb8ce741a311

you can export postmon collection in json format.

save the json in project and run it from there.

also u need to set environment file with url